R/compare_life_satisfaction.R

```diff
@@ -0,0 +1,106 @@
+# SCRIPT TO COMPARE LIFE SATISFACTION ON COUNTRY LEVEL
```

Comment:
I would change the name of this .R file to "plot_lifeSat" because I first thought that this was a different function, and you did the documentation and testing on two different functions. By calling this .R file "plot_lifeSat.R" you make it more clear.